app/src/main/java/org/oppia/app/home/topiclist/TopicListAdapter.kt

```diff
@@ -188,7 +188,8 @@ class TopicListAdapter(
 
       val marginMax = (activity as Context).resources.getDimensionPixelSize(R.dimen.home_margin_max)
 
-      val marginTopBottom = (activity as Context).resources.getDimensionPixelSize(R.dimen.margin_12)
+      val marginTopBottom =
```

Comment:
any specific reason for line change?

Reply:
Yes actully I was getting  Kotlin lint check failing because of this line, So you suggested to reformat the code. https://github.com/oppia/oppia-android/pull/1851#issuecomment-694380789 
After which all the test cases passed.